pkg/virt-launcher/virtwrap/cmd-server/server_test.go

```diff
@@ -193,16 +185,15 @@ var _ = Describe("Virt remote commands", func() {
 		})
 
 		It("client should return disconnected after server stops", func() {
-			err := client.Ping()
-			Expect(err).ToNot(HaveOccurred())
+			Expect(client.Ping()).To(Succeed())
 
 			close(stop)
 			stopped = true
 			time.Sleep(time.Second)
 
 			client.Close()
 
-			err = client.Ping()
+			err := client.Ping()
 			Expect(err).To(HaveOccurred())
```

Comment:
nit: you could inline these 2.

Reply:
I cannot inline them, because the error is also checked on the following line.